Article/Article.js

```diff
@@ -1,29 +1,43 @@
-// Because classes are not hoisted you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
+// Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
-  }
-
-  expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
-
-  }
+    constructor(domElement) {
+        // assign this.domElement to the passed in domElement
+        this.domElement = domElement;
+        console.log(this.domElement);
+        // create a reference to the ".expandButton" class. 
+        this.expandButton = this.domElement.querySelectorAll('.expandButton');
+        // Using your expandButton reference, update the text on your expandButton to say "expand"
+        this.expandButton.forEach((button) => {
+            button.textContent = 'expand';
+        })
+
+        // Set a click handler on the expandButton reference, calling the expandArticle method.
+        this.expandButton.forEach((button) => {
+            button.addEventListener('click', (e) => {
+                this.expandArticle(e);
```

Comment:
i'm wondering why you are passing the click event into this.expandArticle, since expandArticle doesn't use or even look for any parameters at all. 
why not >> 
```
button.addEventListener('click', () =>  this.expandArticle() );
```